src/Core/Installer/Controller/RequirementsController.php

```diff
@@ -16,17 +15,11 @@
 #[Package('framework')]
 class RequirementsController extends InstallerController
 {
-    private readonly string $jwtDir;
```

Comment:
didn't @keulinho do some work on removing JWT already?

Reply:
yep, I parked this for now :)